docs/_panels/github-pull-requests-comments-and-collaboration.md

```diff
@@ -72,4 +72,31 @@ The fields that can be used to aggregate data from both issues and pull requests
 * `issue_id_in_repo`: this works only at repo level as they are unique by repo only. As pull requests are
    issues too, they also contain this id.
 
-[github2_pull_requests]: https://github.com/chaoss/grimoirelab-elk/tree/master/schema/github2_pull_requests.csv
+### Files
+To use this dashboard with your own GrimoireLab deployment you need to:
+* Check [`github2_pull_requests` index][github2_pull_requests-schema] is available on your GrimoireLab instance
+(see [grimoirelab-sirmordred documentation][sirmordred-github2_pull_requests] for details on how to deploy it).
+* Import the following JSON files using [Kidash tool](https://github.com/chaoss/grimoirelab-kidash/).
+
+| [![Index Pattern][ip-icon]][index-pattern] | | [![Dashboard][dash-icon]][dashboard] |
+| :---------: | ---------- | :-------------: |
+| **Index Pattern** | ----- | **Dashboard** |
+
+<br />
+
+#### Command line instructions
+Once you have the data in place, if you need to manually upload the dashboard execute the
+following commands:
+```
+kidash -e https://user:pass@localhost:443/data --import github2_pull_requests-index-pattern.json
+kidash -e https://user:pass@localhost:443/data --import github2_pull_requests_comments_and_collaboration.json
+```
+
+[github2_pull_requests-schema]: https://github.com/chaoss/grimoirelab-elk/blob/master/schema/github2_pull_requests.csv
+[sirmordred-github2_pull_requests]: https://github.com/chaoss/grimoirelab-sirmordred#github2-
+[dash-icon]: ../assets/images/icons/dashboard.png
+[ip-icon]: ../assets/images/icons/file-ruled.png
+[index-pattern]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/github2_pull_requests-index-pattern.json
+[dashboard]: https://raw.githubusercontent.com/chaoss/grimoirelab-sigils/master/json/github2_pull_requests_comments_and_collaboration.json
+
+
```

Comment:
I think this blank line should be removed (This is not important, but I observed most of the modified .md files have 0 or 1 blank lines at the end).

Reply:
Now each and every file should have a single new line at the end.